package.json

```diff
@@ -117,6 +117,7 @@
     "react-router": "^2.2.0",
     "react-router-redux": "^4.0.0",
     "redux": "^3.0.0",
+    "redux-form": "^5.3.1",
```

Comment:
new redux form has better design decisions — it separates redux connect and redux-form
Performance is also significantly improved:
http://redux-form.com/6.0.0-alpha.15/docs/MigrationGuide.md/ (read inversion of control)